src/output/tiles.cpp

```diff
@@ -18,11 +18,11 @@ constexpr uint32_t kMinimumLength = 5;
 constexpr uint32_t kMaximumLength = 1000;
 
 // Local stats for testing
-int count = 0;
+std::unordered_map<uint32_t, uint32_t> count;
 int shortsegs = 0;
 int longsegs = 0;
 int chunks = 0;
-float accum = 0.0f;
+std::unordered_map<uint32_t, float> accum;
```

Comment:
i wouldnt use float here. for large numbers float is really inaccurate. given this map is insanely small you may as well use double.

Reply:
updated